src/main/java/bio/terra/app/utils/startup/StartupInitializer.java

```diff
@@ -14,13 +13,8 @@ private StartupInitializer() {
     }
 
     public static void initialize(ApplicationContext applicationContext) {
-        Migrate migrate = (Migrate)applicationContext.getBean("migrate");
+        // Initialize jobService, stairway, migrate databases, perform recovery
         JobService jobService = (JobService)applicationContext.getBean("jobService");
-
-        logger.info("Migrating all databases");
```

Comment:
Would it be worth moving this logging so it is preserved?

Reply:
I don't think it is needed. There is a lot of logging within the migrate code and it is very visible in the logs when liquibase runs.